templates/patterns/banners/pattern-banner.html.twig

```diff
@@ -0,0 +1,50 @@
+{#
+/**
+ * @file
+ * Default banner component.
+ */
+#}
+
+{% set _link = {'path' : url, 'label' : link_text, icon_position: 'after'} %}
+{% set _centered = true %}
+{% if 'left' in modifier|render %}
+  {% set _centered = false %}
+{% endif %}
+
+{% if 'page' in modifier|render %}
+  {% include '@ecl-twig/ec-component-page-banner/ecl-page-banner.html.twig' with {
```

Comment:
There is no need to define the folder, we have a shorthand version: '@ecl-twig/page-banner'